projects/kit/components/slider/slider.component.ts

```diff
@@ -84,6 +95,31 @@ export class TuiSliderComponent {
         this.el.value = `${newValue}`;
     }
 
+    @Input({
+        alias: 'min',
+        transform: (x: number | string) => coerceNumberProperty(x), // TODO: remove `transform` in v5
```

Comment:
Can it be just `transform: coerceNumberProperty`?

Reply:
It will throw error:

```
Error: node_modules/@angular/cdk/coercion/index.d.ts:30:1 - error NG1010:
Input transform function cannot have multiple signatures
  Value is a reference to 'coerceNumberProperty'.

30 export declare function coerceNumberProperty(value: any): number;
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/@angular/cdk/coercion/index.d.ts:30:25
    30 export declare function coerceNumberProperty(value: any): number;
                               ~~~~~~~~~~~~~~~~~~~~
    Reference is declared here.
```